npc/re/warps/cities/izlude.txt

```diff
@@ -52,6 +52,68 @@ izlude_b,52,172,0	warp	iz04_b	2,2,izlude_in,74,161
 izlude_c,52,172,0	warp	iz04_c	2,2,izlude_in,74,161
 izlude_d,52,172,0	warp	iz04_d	2,2,izlude_in,74,161
 
+//= iz_int(Intro) & int_land(Secluded Island) ================
+iz_int,27,30,0	script	iz_int#room1-1	WARPNPC,2,2,{
+	end;
+
+OnTouch:
+	warp strnpcinfo(1), 51, 30;
```

Comment:
There is no point to use `strnpcinfo(1)` here. `warp "iz_int", 51, 30;`
EDIT: unless we add the duplicate npc of `iz_int` map